rust/connlib/shared/src/messages.rs

```diff
@@ -277,6 +277,7 @@ impl ResourceDescription {
                 dns_a.address != dns_b.address
             }
             (ResourceDescription::Cidr(cidr_a), ResourceDescription::Cidr(cidr_b)) => {
+                println!("{} {}", cidr_a.address, cidr_b.address);
```

Comment:
Either make it a tracing log or remove it :)

Reply:
Oops, leftover from debugging, I will delete